server/services/prisonerService.js

```diff
@@ -70,17 +70,11 @@ function createPrisonerService(nomisClientBuilder) {
     return [merge(coms[0], { deliusId: id.identifierValue })]
   }
 
-  async function getPrisonerImage(imageId, token) {
-    try {
-      logger.info(`getPrisonerImage: ${imageId}`)
+  function getPrisonerImage(imageId, token) {
+    logger.info(`getPrisonerImage: ${imageId}`)
 
-      const nomisClient = nomisClientBuilder(token)
-      const image = await nomisClient.getImageData(imageId)
-      return image
-    } catch (error) {
-      logger.error('Error getting prisoner image')
-      return null
-    }
+    const nomisClient = nomisClientBuilder(token)
```

Comment:
out of interest, why the code change here?

Reply:
The linter highlighted defining image just to return it on the next line and while I was there I noticed that it’s pointless to do so. An async function returns a promise so all we’re doing is resolving a promise just to wrap it in one again. May as well pass the original promise straight through